Documentation/prereqs.md

```diff
@@ -3,5 +3,5 @@
 
 * [Windows Prerequisites](https://docs.microsoft.com/dotnet/core/install/dependencies?pivots=os-windows)
 * [macOS Prerequisites](https://docs.microsoft.com/dotnet/core/install/dependencies?pivots=os-macos)
-* [Linux Prerequisites](https://docs.microsoft.com/dotnet/core/install/dependencies?pivots=os-linux)
+* [Linux Prerequisites](https://docs.microsoft.com/da-dk/dotnet/core/install/linux)
```

Comment:
I don't think the default documentation, which is all in English, should be pointing to a page in Danish.

Reply:
```suggestion
* [Linux Prerequisites](https://docs.microsoft.com/dotnet/core/install/linux)
```